exercises/concept/simple-calculator/.docs/hints.md

```diff
@@ -1,2 +1,8 @@
 # Hints
 
+## General
+
+- Knowing about conditional statements such as `if/else` statements or `switch`
```

Comment:
Conditional statements are statements such as `something && something_else`, not `if..else.end`, this is flow control, though they can use conditional statements.

`switch` is not a thing in Ruby, but `case` is.  And this is not needed for this exercise.

Reply:
Absolutely right ! Thank you @kotp.